graphql/resolvers/dashboardBasic.ts

```diff
@@ -36,89 +38,71 @@ export class BasicDashboardReslover {
     return await pool
       .connect()
       .then(async (client: any) => {
-        client.query("BEGIN")
-        return client
-          .query(`CALL get_dashboard_basic($1, $2, null, null, null);`, [
-            user,
-            runningAvg,
-          ])
-          .then(async () => {
```

Comment:
@ChrisShep98 this is the only part that was fully removed, because we don't have to call the stored proc just to access the cursors anymore. below it's still 3 separate calls to the database, but the difference is we're calling separate functions that return the respective tables now. table and data formats remain completely the same

Reply:
What are the cursors?

Reply:
Check the old stored procedure file that was deleted in this pr. Each of the three tables are returned as refcursors which is why we had to access them in the resolver with FETCH ALL